CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+## How to Contribute as an Outsider
+
+1. Fork the project
+2. Create a feature branch
+3. Make atomic and well thought commits
+4. Submit Pull Request and cc @githubstudent
+  - This is done through the [fork and pull request model](https://help.github.com/articles/using-pull-requests/#fork--pull) 
```

Comment:
Glad i had this linked frmo the text itself. Maybe in a future cleanup PR i can capitalize fork and PR